index.js

```diff
@@ -972,6 +972,22 @@
     return either instanceof Left ? l(either.value) : r(either.value);
   });
 
+  //# maybeToEither :: a -> Maybe b -> Either a b
+  //.
+  //. Converts a Maybe value to an Either value, using the supplied
+  //. parameter as the Left value if the Maybe is Nothing.
+  //.
+  //. ```javascript
+  //. > S.maybeToEither('Expecting an integer', S.parseInt(10, 'xyz'))
```

Comment:
Do you think the `parseInt` is too much for the example here? Seems like most of the examples in these descriptions are very simple, I can just be explicit with `Nothing` and `Just(42)`